components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.publisher/src/main/java/org/wso2/carbon/apimgt/usage/publisher/dto/enums/EVENT_TYPE.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.wso2.carbon.apimgt.usage.publisher.dto.enums;
+
+/**
+ * ENUM to denote event types
+ */
+public enum EVENT_TYPE {
+    SUCCESS,
+    FAULTY,
+    RESOURCE_NOT_FOUND,
+    API_NOT_FOUND,
+    METHOD_NOT_ALLOWED,
```

Comment:
Aren't these all Error categories?

Reply:
Btw, what's the use of this enum?

Reply:
We don't handle security for these types and no authentication context. So we cannot handle it as faulty events we publish. So thought to handle them in separate cases. 
But yes these are faulty events.